client/my-sites/posts/post-controls.jsx

```diff
@@ -167,6 +171,16 @@ module.exports = React.createClass( {
 			}
 		}
 
+		if ( post.status === 'publish' && utils.userCan( 'edit_post', post ) ) {
```

Comment:
This is a huge `render` method and I feel we’re adding even more on top of the ton of code. How can we make this more manageable?